src/Sarif.Viewer.VisualStudio/CodeAnalysisResultManager.cs

```diff
@@ -744,9 +744,13 @@ internal bool TryResolveFilePathFromSolution(string solutionPath, string pathFro
                     return true;
                 }
             }
-            catch
+            catch (Exception ex) when (ex is ArgumentException ||
+                                       ex is IOException ||
+                                       ex is PathTooLongException ||
```

Comment:
>PathTooLongException [](http://example.com/codeflow?start=45&length=20)

PathTooLongException derives from IOException so you don't need to call it out on its own.

Reply:
removed.